src/formatting/package.json

```diff
@@ -15,5 +15,8 @@
   "publishConfig": {
     "access": "public"
   },
+  "dependencies": {
+    "@build-tracker/types": "^1.0.0-alpha.3"
```

Comment:
https://github.com/paularmstrong/build-tracker/blob/ee6e071ef38eabf07a0f88d27bc6a0c9fce95e73/src/formatting/src/index.ts#L4

One of those is an enum that ends up in the runtime.